pkgs/applications/window-managers/hyprwm/hyprland-plugins/hyprsplit.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  meson,
+  fetchFromGitHub,
+  hyprland,
+  ninja,
+  mkHyprlandPlugin,
+}:
+mkHyprlandPlugin hyprland rec {
+  pluginName = "hyprsplit";
+  version = "0.43.0";
+
+  src = fetchFromGitHub {
+    owner = "shezdy";
+    repo = "${pluginName}";
```

```suggestion
    repo = "hyprsplit";
```

always and always use the repo name explicitly.

---

done